src/main/java/thaumicboots/api/IBoots.java

```diff
@@ -19,6 +19,7 @@ public interface IBoots {
     String TAG_MODE_JUMP = "jump";
     String TAG_MODE_SPEED = "speed";
     String TAG_MODE_OMNI = "omni";
+    String TAG_MODE_INERTIA_CANCELLING = "inertiacancelling";
```

Comment:
would you mind changing "TAG_MODE_INERTIA_CANCELLING" to just "TAG_MODE_INERTIA" ?

Just a stylistic choice since all the others are 3 word strings.